STNetwork/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/network/exceptions/NetworkException.kt

```diff
@@ -19,4 +19,12 @@ package com.infomaniak.multiplatform_swisstransfer.network.exceptions
 
 import io.ktor.utils.io.errors.IOException
 
+/**
+ * Thrown when a network-related error occurs, such as connectivity issues or timeouts.
+ *
+ * This exception is used to represent errors that occur during network operations,
+ * typically indicating problems with the connection, DNS resolution, or other network failures.
```

Comment:
Are these two lines really necessary? Because I feel like they don't add much but they double the length of the comment and the time it takes to read and process it